README.md

```diff
@@ -26,7 +26,7 @@ The current compatibility matrix reads as follows:
 
 | `lndclient` git tag          | `lnd` version in `go.mod` | minimum required `lnd` version | 
 | ---------------------------- | ------------------------- | ------------------------------ |
-| [`v0.14.0-5`](https://github.com/lightninglabs/lndclient/blob/v0.14.0-5) | `v0.14.0-beta` | `v0.14.0-beta` |
+| [`v0.14.0-7`](https://github.com/lightninglabs/lndclient/blob/v0.14.0-7) | `v0.14.1-beta` | `v0.14.0-beta` |
```

Comment:
Minimum required `lnd` version should be `lnd @ 4a62b7c`, since somebody running `v0.14.0-beta` wouldn't have this field populated in the rpc.

Reply:
Maybe we should wait for `v0.14.2-beta`?

Reply:
We can, or make a `14-2` branch like @guggero suggested. Happy with either

Reply:
With the branch we don't need to wait for the release necessary. But the README should now have its own entry for the branch with the tag `v0.14.2-0`.

Reply:
cool updated!